React/Modules/RCTUIManager.m

```diff
@@ -1123,6 +1123,12 @@ - (NSDictionary *)customBubblingEventTypes
         @"captured": @"onNavigationCompleteCapture"
       }
     },
+    @"topNavLeftButtonTap": @{
```

Comment:
We should migrate these out of RCTUIManager at some point - they should probably be regular event emitter stuff.